src/resource/user/dto/inputs/update-profile.input.ts

```diff
@@ -3,6 +3,10 @@ import { IsNotEmpty } from 'class-validator';
 
 @InputType()
 export class UpdateProfileInput {
+  @Field()
+  @IsNotEmpty()
+  id: string;
+
   @Field()
   @IsNotEmpty()
   username: string;
```

Comment:
Is this username still in use? can be removed if not. Otherwise, LGTM.

Reply:
Fixed